### ISSUE TITLE
Missing Tabs component

### ISSUE THREAD

**Issue by argosk** [xotahal/react-native-material-ui#32]
First of all a big thanks for this amazing project. I'm going to use it for my next app but I see that missing the [Tabs component](https://material.google.com/components/tabs.html#) It would be nice to have this component, any plans?

**Comment by xotahal** [comment]
Interesting idea. I'm not sure if it's just about style. Do you want to make a pull request? ;)

**Comment by Philip-Nunoo** [comment]
It would be interesting to have this

**Comment by meiraleal** [comment]
I'm going to need this in a few days. I'm going to study the material design guideline and try to implement it and send a PR. Any directions, @xotahal ?

**Comment by xotahal** [comment]
Great @meiraleal 👍  Just try to follow style of another components. There always have to be `style` prop which have a `container`. Then there have to be `getStyles` method that counting the style for a component. And of course - eslint.

Think it won't be hard to implement this component without animation. Animations will be harder. And then it should have some events like - `onTabChange`, etc.

**Comment by xotahal** [comment]
Hey @meiraleal, I just realized that I implemented [bottom navigation](https://material.google.com/components/bottom-navigation.html). I try to make new component here for this bottom navigation and then it probably will be only about position and background color. What do you mean?

This is the result:
<img width="449" alt="screen shot 2016-12-02 at 10 23 09 am" src="https://cloud.githubusercontent.com/assets/3531955/20828762/59009592-b879-11e6-83a6-50634a8b0af6.png">

**Comment by meiraleal** [comment]
I think its good, @xotahal! Are you handling the panels (content) with this component? I'd implement it without content handling, just the events, like onTabChange, and the user handle it.

**Comment by xotahal** [comment]
Yeah, it's the question. For this bottom navigation is ok without handling the content. But for the tab navigation - everyone needs to handle animation to work with "changing" the content due to the tab underline. If you understand me ... We will see, we can do it without handling the content and in the feature we can improve it ;)

**Comment by simsim0709** [comment]
@xotahal Why don't you integrate with [react-native-scrollable-tab-view](https://github.com/skv-headless/react-native-scrollable-tab-view)? It deals with animation already.
I'm currently working on it actually. I can make a pull request if it fits with this library.

**Comment by xotahal** [comment]
@simsim0709 I don't think we need it here. This library is just about styles/material design. And I think your library is more about navigation.

For example, I use `BottomNavigation` component from this library as a custom `tabBarComponent` in react-navigation. And it works in the same way how your `react-native-scrollable-tab-view` works. You know what I mean, don't you?

**Comment by simsim0709** [comment]
@xotahal  Yep. I get it :)

**Comment by Jarrio** [comment]
I agree that those header tabs would be nice. How about implementing [this component](https://github.com/react-native-community/react-native-tab-view) into the project somehow?

**Comment by AleksandarSimikic** [comment]
Any updates regarding this issue ? I still cant implement Tabs from this library.
